src/build.d

```diff
@@ -836,7 +849,10 @@ auto getHostCXX()
         if (model == "32")
             return "dmc";
         else if (model == "64")
+        {
+            buildMsvcDmc();
             return `vcbuild\msvc-dmc`;
```

Comment:
`return buildMsvcDmc()` (otherwise you won't use the built msvc-dmc.exe)

Reply:
I added `msvc-dmd` and `msvc-lib` dependencies for `cxxBackend`.  I think I need `processEnvironment` to run first before those can be built.